src/Pie.js

```diff
@@ -42,15 +42,18 @@ export default class PieChart extends Component {
         bold: true,
         color: '#ECF0F1'
       }
-    },
+    }
   }
 
   color(i) {
-    let color = this.props.color || (this.props.options && this.props.options.color)
-    if (color && !_.isString(color)) color = color.color
-    let pallete = this.props.pallete || (this.props.options && this.props.options.pallete) || Colors.mix(color || '#9ac7f7')
-    return Colors.string(cyclic(pallete, i))
-  }
+    let color = this.props.options.color
```

Comment:
This has reverted a latest master change. The color value should still be pulled from either props.color or props.options.color as in current master.